tests/js/setup.js

```diff
@@ -370,10 +370,20 @@ window.TestStubs = {
     }
   },
 
+  Event: params => {
+    return {
+      ...params,
+      id: '1',
+      message: 'ApiException',
+      groupID: '1',
+      eventID: '12345',
+    };
+  },
+
   Events: () => {
     return [
-      {eventID: '12345', id: '1', message: 'ApiException', groupID: '1'},
-      {eventID: '12346', id: '2', message: 'TestException', groupID: '1'},
+      Event({eventID: '12345', id: '1', message: 'ApiException', groupID: '1'}),
```

Comment:
wait, like https://developer.mozilla.org/en-US/docs/Web/API/Event ?

Reply:
oops i see now

Reply:
i think this was wrong anyways so changed to TestStubs.Event